en/docs/learn/try-device-flow.md

```diff
@@ -0,0 +1,84 @@
+# Try Device Authorization Grant
+
+The Device Authorization Grant is one of the grant types in the OAuth 2.0
+specification. For more information about this grant type, see
+[Device Authorization Grant](../../learn/device-flow-grant).
+
+!!! note "Before you begin" 
+    You must first set up the `Device Flow Demo App` sample webapp.
+    in order to try the following scenario.   
+    
+Step 1:
```

Comment:
please use numbering like this instead of "Step 1" "Step 2" etc. 

1.  Step one content
2. Step two content